README.md

```diff
@@ -17,7 +17,10 @@ If this proposal stands, we plan to extend the [Background Sync](https://wicg.gi
 specification to describe this ability as well, and move the document to the standardization track.
 
 ## Use cases
-  * todo
+  There are two main use cases:
+  1. Web apps can opportunistically update state in the background.
```

Comment:
What sort of state? There's content, but also the application runtime code itself that could be updated. It would be good to mention what _opportunistically_ means, but also why this would be good for the user.

Reply:
Addressed in the updated text.